.ecrc

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Is this file needed?

Reply:
I added it myself in my local to prevent the `node_modules` directory to be added to the files to lint.
Maybe on GH it's not necessary, going to try.

Reply:
Indeed, that file is not necessary on GH because it uses a docker image and the `node_modules` directory doesn't pollute the repository.

Reply:
I removed it.